src/targets/slack.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Bristol
+ * Copyright 2014-2016 Tom Shawver
+ */
+
+'use strict'
+
+const request = require('request-promise')
+
+/**
+ * The Slack target posts to a Slack channel via POST to a webhook.
+ * @param {Object} options Map of options for Slack
+ * @param {string} options.webhook Slack webhook URL
+ * @param {boolean} [options.format=false] Attempt to format the message
+ * @param {boolean} [options.wrapBlock] Wrap log block in code/pre
+ * @param {string} [options.icon_emoji] Slack emoji code for message icon, ex: :boom:
+ * @param {string} [options.username] Username to display
+ * @param {string} [options.channel] The channel to send to in Slack
+ * @param {string} severity Unused
+ * @param {Date} date Unused
+ * @param {string} message The message to be sent to Slack
+ */
+module.exports = (options, severity, date, message) => {
+  let text
+  const nlIndex = message.indexOf('\n')
```

Comment:
One more request! Let's hide this in a conditional so that we're not doing heavy string operations if the user doesn't even have this option enabled.